docs/detections/session-view.asciidoc

```diff
@@ -40,11 +47,11 @@ Events and sessions that you can investigate in Session View have a rectangular
 * On the Alerts page, scroll down to view the Alerts table.
 Look for alerts that have the **Open Session View** button in the **Actions** column:
 [role="screenshot"]
-image::images/session-view-action-icon-detail.png[Detail of the Open Session View icon,width=75%]
+image::images/session-view-action-icon-detail.png[Detail of the Open Session View button,width=75%]
 
 * On the Hosts page (*Explore* -> *Hosts*), select the *Sessions* or the *Events* tab.
-From either of these tabs, click the *Open Session View* icon for an event or session.
-Labeled below are 1) the *Sessions* tab, and 2) the *Open Session View* icon:
+From either of these tabs, click the *Open Session View* button for an event or session.
+Labeled below are *1.* the *Sessions* tab, and *2.* the *Open Session View* button:
```

Comment:
I don't think the number labels are really necessary here, or even the image itself: Elastic's [guidelines](https://docs.elastic.dev/content-design/tech-docs/writing-about-ui#callouts-on-screenshots) ("Callouts on screenshots") suggest labels when an image is "very complex," but this image is pretty simple. The Sessions tab is no more difficult to find than any other tab throughout the app, and the Open Session View icon was already identified in the previous image (session-view-action-icon-detail.png).

If we did want to keep the labels and image, they'd probably look better and more consistent if the labels were formatted like the Session View UI section below (basically a numbered list).